helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -335,7 +335,7 @@ internal fun OrtResult.getRepositoryLicenseFindingCurations(): RepositoryLicense
     repository.nestedRepositories.forEach { (path, vcs) ->
         val pathExcludesForRepository = result.getOrPut(vcs.url) { mutableListOf() }
         curations.forEach { curation ->
-            if (curation.path.startsWith(path)) {
+            if (curation.path.startsWith("$path/")) {
```

Comment:
Can you rewrite the commit message? From the code changes I immediately saw what the problem was, but the commit message left me clueless. Suggestion:

Utils: Fix determining if a path belongs to a nested repository

The check if a path belongs to a nested repository could fail if a file path started with the same string, because it did not consider the file separator. For example, it returned true if a nested repository was located at "a/" and the path was "ab/c".